src/datahike/api.cljc

```diff
@@ -658,17 +759,20 @@
                      :args [@conn]}) ; => #{[\"Alice\" 35] [\"Bob\" 30]}"}
   as-of
   (fn [db time-point]
-    {:pre [(or (int? time-point) (date? time-point))]}
+    {:pre [(or (int? time-point) (spec/date? time-point))]}
```

Comment:
Yes, I'd also say we can drop this. I think this is there mostly for documentation purposes anyway and since this will be done by the specs there isn't a reason anymore to keep it.

Reply:
I have removed the statements in the api namespace now.